src/formula-to-expression.js

```diff
@@ -0,0 +1,17 @@
+import jsep from 'jsep';
+import handleSyntaxErrors from './handle-syntax-errors';
+import astToExpression from './ast-to-expression';
```

Comment:
What's the reasoning for keeping `astToExpression` as a separate file rather than just folding it into this `formulaToExpression` file?

Reply:
Good point ... I'll look closer and think about combining them before merging.